doc/nrf/ug_thingy91.rst

```diff
@@ -205,35 +205,11 @@ Complete the following steps after installing the |nRFVSC|:
 Building and programming on the command line
 ============================================
 
-Complete the :ref:`command-line build setup <build_environment_cli>` before you start building |NCS| projects on the command line.
+.. |cmd_folder_path| replace:: on the nRF9160 SiP component and ``ncs/nrf/applications/connectivity_bridge`` when building the source code for the :ref:`connectivity_bridge` application on the nRF52840 SoC component
```

Comment:
I think there already exists a separate guide for flashing the board controller firmware. It's linked from the «Board controller» section at the top of this document:

    See :ref:`zephyr:nrf9160dk_board_controller_firmware` for detailed information.